hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseOptions.java

```diff
@@ -137,4 +138,28 @@ public static synchronized HbaseOptions instance() {
                     null,
                     ""
             );
+
+    public static final ConfigOption<Integer> HBASE_VERTEX_TABLE_REGION_NUM =
+            new ConfigOption<>(
+                    "hbase.hbase_vertex_table_region_num",
+                    "The number of partitions of the HBase vertex table, it can be set to the number of pre-split regions.",
```

Comment:
We need tell users to keep the partition number **linear correlation**  with the region servers amount, and due to the rowkey will be discretize enough, we should give at least 1X~4X for each region server?

I suggest we add comment for users to modify the value by themselves, and the default region num could consider **3~5** for vertex & **12~20** for edge (consider the edge usually `5X ~ 20X` to vertex)